inc/version_decl

```diff
@@ -1 +1 @@
-   CHARACTER (LEN=10) :: release_version = 'V4.1      '
+   CHARACTER (LEN=10) :: release_version = 'V4.1.1'
```

Comment:
Ming,
Michael suggested to also change the `LEN=10` to `LEN=*`.

Would you make this change, and then run the WRF model and look at the print out. We should see something like:
```
WRF V4.1.1 MODEL
```
Update the commit message that you have conducted this test.

Reply:
@jamiebresch 
... and with a parameter statement, too